client/api/courses.js

```diff
@@ -43,6 +47,14 @@ class CoursesController {
                         label: "Closed Classes"
                     }],
                     selected: null
+                }, {
+                    title: 'Schedule Conflicts',
+                    key: 'shedule',
+                    possibles: [sc, {
```

Comment:
Place that variable's contents here: The reasoning is that the variable is not being used anywhere but here so it should just be right in here.